packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -54,7 +54,7 @@ export interface GraphNGProps extends Themeable2 {
   children?: (builder: UPlotConfigBuilder, alignedFrame: DataFrame) => React.ReactNode;
   prepConfig: (alignedFrame: DataFrame, allFrames: DataFrame[], getTimeRange: () => TimeRange) => UPlotConfigBuilder;
   propsToDiff?: Array<string | PropDiffFn>;
-  preparePlotFrame?: (frames: DataFrame[], dimFields: XYFieldMatchers) => DataFrame;
+  preparePlotFrame?: (frames: DataFrame[], dimFields: XYFieldMatchers) => DataFrame | null;
```

Comment:
@leeoniya how do you feel about this change / modifying this API with option for return value of preparePlotFrame to be `null`?

Reply:
it matches the signature of `preparePlotFrame` util funcs, so should be fine.